File: primer_trimestre/poo/php/ejercicios_php/first/libreria/help.php
<?php

class Help extends Employee {
    private $helpM;
    private $minWage = 1300000;

    public function __construct($helpM) {
        $this->helpM = $helpM;
    }

    public function getHelpM() {
        return $this->helpM;
    }

    public function calculateHelp() {
        if ($this->getBaseSalary() < 2 * $this->minWage) {
            return 0;
        } else {
            return $this->minWage;
        }
    }
}
?>

File: primer_trimestre/poo/java/proyectos_java/ejercicio_1_java/src/help.java
public class help extends Employee {
    
    private int helpM;
    private int minWage = 1300000;
    
    public help( int helpM) {
        this.helpM = helpM;
    }
    public int getHelpM() {
        return helpM;
    }
    public int calculateHelp(){
        if(getBaseSalary() < 2 * this.minWage){
            return 0;
        }
        else {
            return this.minWage;
        }
    }
}
